src/tsdl.ml

```diff
@@ -1514,6 +1545,26 @@ let render_fill_rects r rs =
   let a = CArray.of_list rect rs in
   render_fill_rects r (to_voidp (CArray.start a)) (CArray.length a)
 
+let render_geometry =
+  foreign "SDL_RenderGeometry"
+    (renderer @-> texture @-> ptr void @-> int @-> ptr void @-> int @-> returning zero_to_ok)
+
+let render_geometry ?indices ?texture r vertices =
+  let a1 = CArray.of_list vertex vertices in
```

Comment:
Note that these `of_lists` are going to be rather inefficient. Now I realize that `Tsdl` offers in a lot of cases such list based easy interfaces. But it would be nice do also provide a `_ba` interface as provided [here for example](https://github.com/dbuenzli/tsdl/blob/dcf8b1f917b8521f188f75a4ac6634e8c0a73e6f/src/tsdl.mli#L948-L957). But the issue is that the SDL_Vertex structure is made of a mixture of integers and floats.